src/index.html

```diff
@@ -20,13 +20,13 @@
     <script>
       const scripts = [];
 
-      if (process.env.NODE_ENV === 'development') {
+      if (dev.isRunning) {
         // Dynamically insert the DLL script in development env in the
         // renderer process
         scripts.push('../.erb/dll/renderer.dev.dll.js');
 
         // Dynamically insert the bundled app script in the renderer process
-        const port = process.env.PORT || 1212;
+        const port = dev.port ?? 1212;
```

Comment:
Why change _locical or_ with _nullish coalescing operator_? What if `dev.port` is an empty string or `0` for some reason? In that case we'll end up with the port being an empty string or `0`.

Reply:
Hey clytras,
I changed to nullish coalescing operator because it's behavior is a lot more predictable than ``||`` which will *short circuit* to right value when it's ``0`` or an empty string as you stated, instead of simply just checking for ``null`` or ``undefined``. Thus, I try to avoid using ``||`` and replace it with ``??`` whenever I see it because it results in less headache for long run when cases like that occur.
Though, I really don't think my change here would have altered the behavior anyway unless one really wants to have either an empty string or ``0`` as their port.

Reply:
It will not alter the default behavior but in a long term it will lead unexperienced and _naive_ developers getting stuck to why things aren't working after some mistaken changes they or some other have made (_or even session environment variables lost/altered_); thus that will may lead to useless issue submissions regarding _"can't start electron"_ and related to port ending being `0` or an empty string.

I suggest we leave these defaults assingments with `||` operator. Logical/assignment `OR` operator is pretty straightforward on its behaviour and in assigmnet, that is _"continue with the next operand if current operand is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)"_, which is what we need here to eliminate the possibility of `port` ending up being `0` or en empty string.

Reply:
Hey,
I have reverted the following change according to your feedback. Please let me know if you have any other suggestions about this PR.

Thanks~